### ISSUE TITLE
command "sh" produced invalid JSON: 

### ISSUE THREAD

**Issue by raghuck** [matti/terraform-shell-resource#14]
Hi,

I  am getting this error. Could someone please let me know what might be the issue

data.external.exitstatus: Refreshing state...
data.external.stderr: Refreshing state...
data.external.stdout: Refreshing state...

Error: Error applying plan:

1 error occurred:
	* data.external.stdout: data.external.stdout: command "sh" produced invalid JSON: invalid character 'A' after object key:value pair

I get the output when I run the read.sh manually
sh read.sh stdout.1150458593394434125 
{"content": ""ACTIVE""}

**Comment by matti** [comment]
while someone changes this to use new terraform 0.12 features can you make your command to return the data without " or escape "

**Comment by matti** [comment]
@raghuck couldn't reproduce this with 0.12 - what kind of command are you running? How can I reproduce this.

**Comment by anoff** [comment]
having the same issue

```
Terraform v0.12.3
+ provider.azuread v0.3.1
+ provider.azurerm v1.31.0
+ provider.external v1.2.0
+ provider.null v2.1.2
```

The command I was running is `command     = "az storage account generate-sas --permissions acdlpruw --resource-types sco --services b --https-only --expiry `date -d 12months '+%Y-%m-%dT%H:%MZ'` --ids ${azurerm_storage_account.logs.id}"` which returns a string like this `"se=2020-07-09T13%3A55Z&sp=rwdlacup&spr=https&sv=2018-03-28&ss=b&srt=sco&sig=ycNqUMGsglLecLSW2tTgD2rvQTg8a5edB1Q%3D"` that gets successfully written into the stdout file but apparently reading it fails.

**Comment by matti** [comment]
This might now be fixed with - reading of files uses terraform 0.12
